tests/www/test_app.py

```diff
@@ -240,7 +240,8 @@ def test_should_set_permanent_session_timeout(self):
     @conf_vars({('webserver', 'cookie_samesite'): ''})
     @dont_initialize_flask_app_submodules
     def test_correct_default_is_set_for_cookie_samesite(self):
-        app = application.cached_app(testing=True)
+        with pytest.deprecated_call():
+            app = application.cached_app(testing=True)
```

Comment:
What is giving deprecation warnings here?

Reply:
The default value of `cookie_samesite` was an empty string, but was changed to `Lax` in 2.0. Setting the config to an empty string now emits a deprecation warning, and the value is automatically corrected to `Lax`. I’ll add a note as a code comment.